test/unit/appender/standard.js

```diff
@@ -96,5 +96,19 @@ describe('Canadarm.Appender', function () {
         assert.equal(actualAttributes.scriptURL, 'fake-src');
       });
     });
+
+    describe('works without getElementsByTagName', function() {
```

Comment:
Although this describe statement is accurate, I'd prefer it if it said something along the lines of "in a Node context" since that's the real problem we're making sure to fix.